src/bindings/python/tests/test_runtime/test_tensor.py

```diff
@@ -640,3 +640,23 @@ def test_copy_and_deepcopy(copy_func, should_share_data):
         assert tensor_copy.data[0, 0] == outlier
     else:
         assert tensor_copy.data[0, 0] == value
+
+
+# supported dtypes by Pillow
+@pytest.mark.parametrize(("numpy_dtype", "shape"), [
+                         (np.float32, (224, 224)),
+                         (np.int32, (224, 224)),
+                         (np.uint8, (224, 224, 3)),
+                         (np.uint16, (224, 224)),],)
+def test_tensor_from_pillow(numpy_dtype, shape):
+    from PIL import Image
+
+    arr = generate_image(shape, numpy_dtype)
+    img = Image.fromarray(arr)
```

Comment:
What will happen when the user will try to pass such `img` to `InferRequest.infer()`? How data dispatch will behave?

Reply:
It won't work right now, but it's possible to pass tensor to infer.

Reply:
Thanks for answer. I didn't know that adding `py::module::import("PIL.Image").attr("Image")))` in cpp file does not add `pillow` as package dependency